packages/honeycomb-opentelemetry-web/README.md

```diff
@@ -133,6 +134,23 @@ Your OpenTelemetry Collector can send the traces on to Honeycomb, and your API k
 }
 ```
 
+### Context Management
+OpenTelemetry uses the concept of a [Context Manager](https://opentelemetry.io/docs/languages/js/context/#context-manager) to store propagate global span context through your system. OpenTelemetry provides a context manager for browser instrumentation based on the [Zone.js](https://github.com/angular/angular/tree/main/packages/zone.js) library to track global context across asynchronous execution threads. This context manager can be plugged into this instrumentation like so:
+
+```js
+import { ZoneContextManager } from '@opentelemetry/context-zone';
+
+const sdk = new HoneycombWebSDK({
+  // other config options omitted...
+  contextManager: new ZoneContextManager()
+});
+sdk.start();
+```
+
+Zone.js has known limitations with async/await code, and [requires](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-context-zone-peer-dep#installation) your code to be transpiled down to ES2015. It also may carry a performance penalty.
+
+For these reasons, we do not enable ZoneContextManager by default.
```

Comment:
This is the primary change, the rest is reformatting the tables to be easier to read in plain text.